packages/x-components/src/views/home/Home.vue

```diff
@@ -390,8 +390,9 @@
 
                           <template #related-prompts-group="{ item: { relatedPrompts } }">
                             <RelatedPrompt
-                              class="x-flex x-flex-col x-pt-24"
+                              class="x-bg-neutral-10 x-flex x-flex-col x-gap-16 x-pt-24"
                               :relatedPrompt="relatedPrompts[0]"
+                              nextQueryButtonClass="x-button x-button-lead x-button-sm x-button-outlined"
```

Comment:
Hi, IMO the `x-button` would rely on the component itself, the rest of them ok here